pkg/kubelet/dockertools/manager.go

```diff
@@ -1190,6 +1192,32 @@ func (dm *DockerManager) PortForward(pod *kubecontainer.Pod, port uint16, stream
 	return command.Run()
 }
 
+// Get the IP address of a container's interface using nsenter
+func (dm *DockerManager) GetContainerIP(containerID, interfaceName string) (string, error) {
```

Comment:
> For future fixup: Could this just be private to the CNI plugin?

Yeah, it probably could be.  I think Rajat just put in the docker stuff for symmetry, but I think in the future this code will go away when we get the IP from AddNetwork or add a STATUS call to the CNI spec.